font-codegen/src/lib.rs

```diff
@@ -87,6 +88,7 @@ pub(crate) fn generate_parse_module(items: &Items) -> Result<proc_macro2::TokenS
             Item::Format(item) => table::generate_format_group(item)?,
             Item::RawEnum(item) => flags_enums::generate_raw_enum(item),
             Item::Flags(item) => flags_enums::generate_flags(item),
+            Item::Extern(..) => TokenStream::new(),
```

Comment:
we can take advantage of the fact that `TokenStream` impls `Default` to write,

```suggestion
            Item::Extern(..) => Default::default(),
```

which... I mean honestly they're both fine. This saves us one line of importing, though!